docs/config.rst

```diff
@@ -1025,12 +1025,51 @@ For example, given this config:
 
 You could enable ``ignore_errors`` by running::
```

Comment:
Probably shouldn't keep the `::` here.

Reply:
No, the code block should handle that. `::` is alternative to `code-block` without specifying the syntax.

Reply:
Right, but you can't have both. You can have something like this:

``` rst
This is some code (note the double colon)::

    Here is my code!
```

_**Or**_ you can have something like this:

``` rst

This is some code (note the single colon):

.. code-block:: lang

    Here is my code!
```

But this won't work (notice the double colon after the text _**and**_ the use of `code-block`:

``` rst
This is some code (note the double colon)::

.. code-block:: lang

    Here is my code!
```